pkg/cli/admin/verifyimagesignature/verify-signature.go

```diff
@@ -261,7 +261,9 @@ func (o *VerifyImageSignatureOptions) getImageManifest(img *imagev1.Image) ([]by
 			registryURL.Scheme = ""
 		}
 	}
-	return getImageManifestByIDFromRegistry(registryURL, parsed.RepositoryName(), img.Name, o.CurrentUser, o.CurrentUserToken, o.Insecure)
+	// when using in-cluster auth, the username is just user + token, compare
+	// https://github.com/openshift/oc/blob/9f54c1d4f68c8530ac9466c655a4e55eb04a1459/pkg/cli/registry/login/login.go#L208
```

Comment:
Check this URL, this answers the question why `user` is being used as a username.

Reply:
so, as long as the token is in use, then "user" is appropriate?  that's what it looks like, but I've been having a hard time verifying/reproducing this issue :(

Reply:
I see the `getImageManifestByIDFromRegistry` method is always passed the `o.CurrentUserToken` so based on that, and looking at how we force the "user" in that case with registry login code, this makes sense, but still, I haven't been able to reproduce or confirm this.